Shopipy.ProductManagement/Controllers/ProductController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Shopipy.Persistence.Models;
+using Shopipy.ProductManagement.DTOs;
+using Shopipy.Shared;
+using Shopipy.Shared.DTOs;
+using Shopipy.Shared.Services;
+
+namespace Shopipy.ProductManagement.Controllers;
+
+[Route("Product/{businessId}/products")]
+[ApiController]
```

Comment:
Put `[Authorize]` on the top, everyone will be able to access otherwise (applies to other controllers)